django_configuration_management/cli.py

```diff
@@ -85,3 +95,113 @@ def reencrypt(environment, new_key):
     print(
         f"config-{environment}.yaml has been re-encrypted. Please be sure to update your .env-{environment} with the new key you used!"
     )
+
+
+@click.command("github_secrets")
+def github_secrets():
+    """
+    1. Get github public auth key from local env
+    2. Get current repo name based on current dir
+    3. Get repo public key
+    4. Loop through `config-github.json` file and:
+        - Ask to provide secret value for specific key
+        - Encrypt provided secret
+        - Save to dictionary
+    5. Push secrets to github repo
+    """
+    key = "GITHUB_ACCESS_TOKEN"
+    github_access_token = os.getenv(key)
+    owner_repo_array = owner_and_repo()
+    owner = owner_repo_array[0]
+    repo_name = owner_repo_array[1]
+
+    # Get a repository public key
+    headers = {"Authorization": "token " + github_access_token}
+    repo_public_key_string = requests.get(
+        f"https://api.github.com/repos/{owner}/{repo_name}/actions/secrets/public-key",
+        headers=headers,
+    ).text
+    repo_public_key_dict = json.loads(repo_public_key_string)
+    repo_key = repo_public_key_dict["key"]
+    key_id = repo_public_key_dict["key_id"]
+
+    github_config_file = open(
+        "config-github.json",
+    )
+    github_config = json.load(github_config_file)
+    user_provided_secrets = {}
+
+    # Prompt user for secret and encrypt
+    for key in github_config:
+        prompt = f"Please provide the secret for key: {key}"
+        repo_secret = input(prompt + " (or `next`):\n")
+        if repo_secret == "next":
+            continue
+        encrypted_secret = encrypt_secret(repo_key, repo_secret)
+        user_provided_secrets[key] = encrypted_secret
+
+    push_secrets_to_github(
+        owner, repo_name, key_id, user_provided_secrets, github_access_token
+    )
+
+
+def owner_and_repo():
```

Comment:
could all of these helper methods be moved to their own file? something like `github_utils.py` or whatever the naming convention this lib uses.